tests/FSharp.Compiler.ComponentTests/Language/ComputationExpressionTests.fs

```diff
@@ -23,6 +23,78 @@ let x = lb {1; 2;}
         |> shouldSucceed
         |> ignore
 
+    [<Fact>]
+    let ``Allow CE return and type annotations to play well together without needing parentheses``() =
```

Comment:
Please add v9 tests for others (return!,yield,yield!).
I think they are not implemented the same (since they go via different rules in the code now)

Reply:
Done